pkg/gce-cloud-provider/compute/gce-compute.go

```diff
@@ -288,6 +297,66 @@ func (cloud *CloudProvider) insertRegionalDisk(ctx context.Context, volKey *meta
 	return nil
 }
 
+func (cloud *CloudProvider) insertRegionalAlphaDisk(ctx context.Context, volKey *meta.Key, diskType string, capBytes int64, capacityRange *csi.CapacityRange, replicaZones []string, snapshotID, diskEncryptionKmsKey string, multiWriter bool) error {
+	diskToCreateAlpha := &computealpha.Disk{
+		Name:        volKey.Name,
+		SizeGb:      common.BytesToGb(capBytes),
+		Description: "Regional disk created by GCE-PD CSI Driver",
+		Type:        cloud.GetDiskTypeURI(volKey, diskType),
+		MultiWriter: multiWriter,
+	}
+	if snapshotID != "" {
+		diskToCreateAlpha.SourceSnapshot = snapshotID
+	}
+	if len(replicaZones) != 0 {
+		diskToCreateAlpha.ReplicaZones = replicaZones
+	}
+	if diskEncryptionKmsKey != "" {
+		diskToCreateAlpha.DiskEncryptionKey = &computealpha.CustomerEncryptionKey{
+			KmsKeyName: diskEncryptionKmsKey,
+		}
+	}
+
+	insertOp, err := cloud.alphaService.RegionDisks.Insert(cloud.project, volKey.Region, diskToCreateAlpha).Context(ctx).Do()
+	if err != nil {
+		if IsGCEError(err, "alreadyExists") {
+			disk, err := cloud.GetDisk(ctx, volKey)
```

Comment:
does `GetDisk` need to be changed as well? If we are to get a disk that is multi-writer what happens (since it is only on the alpha struct)?

Reply:
you may be able to leverage `CloudDisk` here (originally created for this different api problem for RePD)

Reply:
Yes, GetDisk has to be updated. CloudDisk looks to be the way to go.